controllers/case-controller.js

```diff
@@ -26,16 +26,20 @@ const CaseController = Case => {
   };
 
   const listCases = async query => {
+    const limit = query.limit && parseInt(query.limit, 10);
```

Comment:
One principle of API design that I've learned the hard way is that we should always have limits.  A 'rogue' client should never be able to take us down by asking for, say, one million results.  So maybe while we're here, we make this something like `const limit = parseInt(query.limit, 10) || 1000;` ?

Reply:
Right! Should we choose the minimum between `query.limit` and 1000, or whatever number the user sends in the query will be fine?